_posts/2017-09-19-ops-s1-swarm-intro.markdown

```diff
@@ -197,8 +197,8 @@ What is a stack?
 
 {:.quiz}
 A stack can:
-- (x) be deployed from the commandline
-- (x) use the compose file format to deploy
-- ( ) run a Dockerfile
-- ( ) be used to manage your hosts
-- (x) be used to manage services over multiple nodes
+- \[x\] be deployed from the commandline
```

Comment:
Hi Marcos,

I tried achieving the goal without backslash escapes first too, but [it makes the whole quiz disappear for me](https://pasteboard.co/HYzyess.png).
My best guess is that [ and ] characters have special meaning in markdown and get removed prior quiz.js gets a chance to do it's job. Also [Markdown cheatsheet](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf) suggests to backslash escape [ ] characters.

I believe they're necessary, otherwise wouldn't have suggested them in the first place.